how-to/integrate-with-bloomberg-basic/README.md

```diff
@@ -0,0 +1,81 @@
+![OpenFin Workspace Example Application -- Integrate With Bloomberg Basic](../../assets/OpenFin-Workspace-Starter.png)
+
+> **_:information_source: OpenFin Workspace:_** [OpenFin Workspace](https://www.openfin.co/workspace/) is a commercial product and this repo is for evaluation purposes (See [LICENSE.MD](LICENSE.MD)). Use of the OpenFin Container and OpenFin Workspace components is only granted pursuant to a license from OpenFin (see [manifest](public/manifest.fin.json)). Please [**contact us**](https://www.openfin.co/workspace/poc/) if you would like to request a developer evaluation key or to discuss a production license.
+> OpenFin Workspace is currently **only supported on Windows** although you can run the sample on a Mac for development purposes.
+
+# Integrate With Bloomberg (Basic)
+
+## Pre-requisites
+
+- Bloomberg Terminal software installed, running and with a user logged in. Software can be acquired from here: <https://www.bloomberg.com/professional/support/software-updates/>
+- A domain that is set uo with Bloomberg IT so that the BBG terminal will accept requests originating from that domain
+
+## Notes about this sample
+
+This application you are about to install is a simple example of connecting to a Bloomberg Terminal.  
+ When a connection is established between your OpenFin app and the Bloomberg Terminal, the Terminal reacts to Interop events and shares the appropriate context. The reaction depends on the type of context. You can configure contexts and intents, but we provide a default configuration for the most common contexts and intents.  
+ The default configuration always directs interop actions to Terminal panel 1 and maps context types and intents to Terminal functions as follows:
+
+| Intent Name    | ContentType     | Terminal Function |
+| -------------- | --------------- | ----------------- |
+| ViewChart      | fdc3.instrument | GP                |
+| ViewContact    | fdc3.contact    | BIO               |
+| ViewInstrument | fdc3.instrument | DES               |
+| ViewQuote      | fdc3.instrument | Q                 |
+
+This example assumes you have already [set up your development environment](https://developers.openfin.co/of-docs/docs/set-up-your-dev-environment)
+
+## Running the Sample
+
+To run this sample you can:
+
+- Clone this repo and follow the instructions below. This will let you customize the sample to learn more about our APIs.
+
+## Getting Started
+
+1. Install dependencies and build the code. Note that these examples assume you are in the sub-directory for the example.
+
+```shell
+npm run setup
+```
+
+2. Optional (if you wish to pin the version of OpenFin Workspace to version 11.0.0 and you are on Windows) - Set Windows registry key for [Desktop Owner Settings](https://developers.openfin.co/docs/desktop-owner-settings).
```

Comment:
This needs to say version 12.0.0 as that is now the main branch instead of 11.0.0.